.travis.yml

```diff
@@ -26,19 +26,20 @@ matrix:
         - "3.6"
 
       before_install:
-        - "sudo service postgresql stop"
+        - sudo service postgresql stop
         - ./docker-inspire up -d
-        - "travis_retry pip install --upgrade pip setuptools py pipenv"
-        - "travis_retry pip install twine wheel coveralls"
-
+        - curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python
+        - source $HOME/.poetry/env
+        - travis_retry poetry install
```

Comment:
this is not needed, we had it to install ```setuptools`` ``py`` ``pipenv`` 

```python
pip install --upgrade pip setuptools py pipenv"
```